RecoLocalMuon/GEMRecHit/test/runGEMRecHitProducer_cfg.py

```diff
@@ -42,6 +42,14 @@
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
```

Comment:
Hi Slava
ok then i will not complicate my life by splitting the PR. 
Thanks a lot
greets
Piet